<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttachmentProperties</class>
 <widget class="QWidget" name="AttachmentProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>526</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>8</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelTexture">
       <property name="text">
        <string>Texture</string>
       </property>
       <property name="indent">
        <number>44</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ReferenceComboBox" name="texture"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelLevel">
       <property name="text">
        <string>Level</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="level"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelLayer">
       <property name="text">
        <string>Layer</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QWidget" name="layerWidget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="layered">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="layer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelBlendColorEq">
       <property name="text">
        <string>Blend Color Eq.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="DataComboBox" name="blendColorEq"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelBlendColorSource">
       <property name="text">
        <string>&lt;html&gt;Source Color &amp;#215;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="DataComboBox" name="blendColorSource"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelBlendColorDest">
       <property name="text">
        <string>&lt;html&gt;Dest. Color &amp;#215;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="DataComboBox" name="blendColorDest"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelBlendAlphaEq">
       <property name="text">
        <string>Blend Alpha Eq.</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="DataComboBox" name="blendAlphaEq"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelBlendAlphaSource">
       <property name="text">
        <string>&lt;html&gt;Source Alpha &amp;#215;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="DataComboBox" name="blendAlphaSource"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="labelBlendAlphaDest">
       <property name="text">
        <string>&lt;html&gt;Dest. Alpha &amp;#215;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="DataComboBox" name="blendAlphaDest"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="labelColorWriteMask">
       <property name="text">
        <string>Write RGBA</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="ColorMask" name="colorWriteMask" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabDepthStencil">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideRight</enum>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tabStencilBack">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <attribute name="title">
       <string>Stencil Back</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QFormLayout" name="formLayout3">
         <property name="labelAlignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="horizontalSpacing">
          <number>8</number>
         </property>
         <property name="verticalSpacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="labelStencilBackReference">
           <property name="text">
            <string>Reference</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="stencilBackReference">
           <property name="minimum">
            <number>-128</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelStencilBackReadMask">
           <property name="text">
            <string>Read Mask</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelStencilBackComparisonFunc">
           <property name="text">
            <string>Stencil Function</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="stencilBackReadMask">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="DataComboBox" name="stencilBackComparisonFunc"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelStencilBackFailOp">
           <property name="text">
            <string>Stencil Fail Op.</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="DataComboBox" name="stencilBackFailOp"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labelStencilBackDepthFailOp">
           <property name="text">
            <string>Depth Fail Op.</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="DataComboBox" name="stencilBackDepthFailOp"/>
         </item>
         <item row="5" column="1">
          <widget class="DataComboBox" name="stencilBackDepthPassOp"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="labelStencilBackDepthPassOp">
           <property name="text">
            <string>Depth Pass Op.</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="labelStencilBackWriteMask">
           <property name="text">
            <string>Write Mask</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QSpinBox" name="stencilBackWriteMask">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabStencilFront">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <attribute name="title">
       <string>Stencil Front</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QFormLayout" name="formLayout2">
         <property name="labelAlignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="horizontalSpacing">
          <number>8</number>
         </property>
         <property name="verticalSpacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="labelStencilReference">
           <property name="text">
            <string>Reference</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="stencilReference">
           <property name="minimum">
            <number>-128</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelStencilReadMask">
           <property name="text">
            <string>Read Mask</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelStencilComparisonFunc">
           <property name="text">
            <string>Stencil Function</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="stencilReadMask">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="DataComboBox" name="stencilComparisonFunc"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelStencilFailOp">
           <property name="text">
            <string>Stencil Fail Op.</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="DataComboBox" name="stencilFailOp"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labelStencilDepthFailOp">
           <property name="text">
            <string>Depth Fail Op.</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="DataComboBox" name="stencilDepthFailOp"/>
         </item>
         <item row="5" column="1">
          <widget class="DataComboBox" name="stencilDepthPassOp"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="labelStencilDepthPassOp">
           <property name="text">
            <string>Depth Pass Op.</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="labelStencilWriteMask">
           <property name="text">
            <string>Write Mask</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QSpinBox" name="stencilWriteMask">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDepth">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <attribute name="title">
       <string>Depth</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QFormLayout" name="formLayout4">
         <property name="labelAlignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="horizontalSpacing">
          <number>8</number>
         </property>
         <property name="verticalSpacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="labelDepthComparisonFunc">
           <property name="text">
            <string>Depth Function</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="DataComboBox" name="depthComparisonFunc"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelDepthOffsetSlope">
           <property name="text">
            <string>Offset Slope</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="depthOffsetSlope">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>-1000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.001000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelDepthOffsetConstant">
           <property name="text">
            <string>Offset Constant</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelDepthClamp">
           <property name="text">
            <string>Depth Clamp</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="depthClamp">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labelDepthWrite">
           <property name="text">
            <string>Write Depth</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="depthWrite">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="depthOffsetConstant">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>-1000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataComboBox</class>
   <extends>QComboBox</extends>
   <header>session/DataComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ReferenceComboBox</class>
   <extends>QComboBox</extends>
   <header>session/ReferenceComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ColorMask</class>
   <extends>QWidget</extends>
   <header>session/ColorMask.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
